build/goTranspiler.ts

```diff
@@ -1563,13 +1559,12 @@ func (this *${className}) Init(userConfig map[string]interface{}) {
             const go = this.transpiler.transpileGoByPath(tsFile);
             let content = go.content;
             content = this.regexAll (content, [
-                [/new ccxt.Exchange.+\n.+\n.+/gm, 'ccxt.NewExchange()' ],
+                [/(\w+) := new ccxt\.Exchange\(([\S\s]+?)\)/gm, '$1 := ccxt.NewExchange().(*ccxt.Exchange); $1.InitParent($2, map[string]interface{}{}, $1)' ],
```

Comment:
we didn't transpile arguments, so from now and onwards:
```
aa = ccxt.Exchange ( {...} );
```
would be transpiled in GO: 
```
aa = ccxt.NewExchange();  aa.InitParent (map[string]interface{}{ ...});
```